src/components/Article.jsx

```diff
@@ -0,0 +1,18 @@
+const Article = ({ article }) => {
+  const { article_img_url, author, comment_count, title, topic, votes } = article;
+  return (
+    <article>
+      <img src={article_img_url} alt="article-image" />
+      <h5>{title}</h5>
+      <div>
```

Comment:
We want to avoid using divs unless we absolutely have to, could we choose better, more semantic html tags here?